File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string str1, string str2) {
    string answer = "";
    int size = str1.size();

    for (int i = 0; i < size; i++) {
        answer.push_back(str1[i]);
        answer.push_back(str2[i]);
    }

    return answer;
}

File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.swift
import Foundation

func solution(_ str1:String, _ str2:String) -> String {
    
    var str1 = str1
    var str2 = str2
    
    var result = ""
    
    while !str1.isEmpty && !str2.isEmpty {
        result.append(str1.removeFirst())
        result.append(str2.removeFirst())
    }
    
    
    return result
}
